tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_db_inode_check.py

```diff
@@ -0,0 +1,133 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import shutil
+import pytest
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import create_file, generate_params, check_time_travel, callback_detect_modified_event, \
+                              LOG_FILE_PATH, REGULAR, callback_detect_modified_event_with_inode_mtime, \
+                              detect_initial_scan
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools.file import truncate_file
+
+# Marks
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0)]
+
+# Variables
+
+monitored_folder = os.path.join(PREFIX, 'testdir1')
+test_directories = [monitored_folder]
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_check_inodes.yaml')
+file_list = ['file1', 'file2', 'file3', 'file4', 'file5', 'file6', 'file7', 'file8', 'file9', 'file10']
+
+# configurations
+
+monitoring_modes = ['scheduled']
+
+conf_params = {'TEST_DIRECTORIES': test_directories, 'MODULE_NAME': __name__}
+p, m = generate_params(extra_params=conf_params, modes=monitoring_modes,
```

Comment:
We're trying to avoid variables named with a single letter. Could you use params and metadata or something like this instead?